### ISSUE TITLE
Warning: Could not get charToByteConverterClass!

### ISSUE THREAD

**Issue by pawlowska** [openspim/micromanager#5]
The (up to date as of today) openspim version of Fiji throws the "Warning: Could not get charToByteConverterClass!" in the Java console at seemingly random times. I tried a clean Fiji version from Fiji website and I didn't have this warning there. I hope this is the right place to post it.

**Comment by dscho** [comment]
It looks as if this warning comes from xalan: https://github.com/zxlooong/jdk14219/blob/b9422ce246ee463a24e55f3e91797b33862c9d4c/org/apache/xalan/serialize/Encodings.java#L162

Do you know of any specifics how to trigger this warning?

(I could also imagine that a recent Java version simply removed the `sun.io.CharToByteConverter#getConverter(java.lang.String)` method... So either a downgrade of Java or an upgrade of xalan might be required to make this warning go away.)

**Comment by pawlowska** [comment]
I'm not exactly sure what triggers the warning, there is only this one line in the console. I'll try to pay more attention. Dylan Windell also mentioned this http://openspim.org/pipermail/openspim/2015-June/000756.html so at least it's not specific to my computer.

**Comment by pawlowska** [comment]
one of the situations it pops up is after updating, more or less at the same time with the message "update finished, please restart fiji"

**Comment by dscho** [comment]
Hmm. If I call the _Find Jar For Class_ helper (via `Ctrl+L`), my Fiji/OpenSPIM claims that the class `org.apache.xalan.serialize.Encodings` is not even found. Is that the same for you?

**Comment by pawlowska** [comment]
Yes, it is, i.e. class not found. So it must be a different one then?

Incidentally, I was doing acquisition right now and the console warning popped up right after it was finished.

**Comment by HFLinfeng** [comment]
perhaps,your other code use 3.15 and your pom has 3.15!while other place need 3.17!i'm very sad that i'm too!try to search better method
